fem/pfespace.cpp

```diff
@@ -1549,16 +1553,19 @@ void ParFiniteElementSpace::UnpackDof(int dof,
       dof -= nedofs;
       if (dof < nfdofs) // regular face
       {
-         if (fdofs) // have mixed faces
-         {
-            index = bisect(fdofs+1, mesh->GetNFaces(), dof);
-            edof = dof - fdofs[index];
-         }
-         else // uniform faces
+         if (uni_fdof >= 0) // uniform faces
          {
             int nf = fec->DofForGeometry(pncmesh->GetFaceGeometry(0));
             index = dof / nf, edof = dof % nf;
          }
+         else // mixed faces or var-order space
+         {
+            const Table &table = var_face_dofs;
+            MFEM_ASSERT(table.Size(), "");
```

Comment:
Should we give a more meaningful message?

Reply:
If you insist... I just can't think of anything better than "internal error" or similar. This is just a routine check that the table is not empty, which the following code assumes.